### PR TITLE
Add KukretiShubham

### LINKED ISSUE #648: Add Shubham Kukreti
screencapture-github-KukretiShubham-2021-08-15-19_23_44

### PR DESCRIPTION
**Pull request by KukretiShubham**
Resolve #648

### CODE REVIEW

**Comment by CodingSpecies** [comment]
@KukretiShubham, I think you should create a pull request and you should add your image at the bottom of the members.json file. thank you.

**Comment by KukretiShubham** [comment]
Isn't it a pull request?🤔

**Comment by CodingSpecies** [comment]
When I press your commit, it takes me to your issue

**Comment by KukretiShubham** [comment]
This is because I gave update message while committing as a reference to issue #648 .

**Comment by CodingSpecies** [comment]
Ok, thank you, sorry, didn't see that, just saw your commit, LGTM.

**Comment by KukretiShubham** [comment]
Don't be sorry. Mistakes make us learn.

**Review by deephunt3r — APPROVED** [review]
LGTM 🚀

File: data.json [hunk 1]
```diff
@@ -1264,5 +1264,11 @@
     "githubUsername": "FayasNoushad",
     "imageUrl": "https://user-images.githubusercontent.com/76828314/129467637-18e9f157-4038-4aed-83e7-7d55e239c03f.jpg",
     "issueNumber": 646
+  },
+  {
+    "name": "Shubham Kukreti",
+    "githubUsername": "KukretiShubham",
+    "imageUrl": "https://user-images.githubusercontent.com/57281769/129481284-fe84223a-f3e4-41d8-9460-009254407e37.png",
```
Comment (Panquesito7) [inline data.json:1271]: I suggest more of an image like this.

image

```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/51391473/129485691-4a1249b6-3203-4f15-ac8c-654a67d50ee4.png",
```

**Review by Panquesito7 — COMMENTED** [review]
Just a suggestion. It doesn't need to be accepted. 🙂

File: data.json (hunk 1 above)
Comment (schmelto) [reply data.json:1271]: Like this suggestion I would recommend going with something like this

**Review by schmelto — COMMENTED** [review]
Lgtm maybe only change the image as showed in https://github.com/EddieHubCommunity/awesome-github-profiles/pull/649#discussion_r689114256

**Review by schmelto — CHANGES_REQUESTED** [review]
See comment above

**Review by Panquesito7 — APPROVED** [review]
LGTM. Thank you for your contribution! 😄👍🎉

**Review by schmelto — APPROVED** [review]

**Comment by schmelto** [comment]
Thanks for your contribution 🚀

**Pull request merged**